src/style.css

```diff
@@ -14,6 +14,10 @@ body {
   border: none;
 }
 
+input {
+  background-color: transparent;
```

Comment:
- [optional] kindly use a friendly color when editing the task on the to-do list. The yellow color is too bright and not user friendly. you can use a lighter color. nice work